data/templates/esx.rackhdcallback

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+# esx       callback to rackhd post installation API hook
+# description: calls back to rackhd post installation API hook
+
+echo "Attempting to call back to RackHD ESX installer"
+
+# *sigh*, busybox shell does not support {1..30}. Retry 30 times with 1 second
+# sleep in between.
+for retry in $(awk 'BEGIN { for ( i=0; i<30; i++ ) { print i; } }');
+do
+    wget -s http://<%=server%>:<%=port%>/api/common/templates/<%=completionUri%>
```

Comment:
I made a mistake, the static file request will also publish to the AMQP, so if we change the completionUri to a file in the OS repository, it will work.

The disadvantage is that the completionUri will vary with OS, even sometime with OS version.